Form/Type/BooleanType.php

```diff
@@ -81,7 +81,9 @@ public function configureOptions(OptionsResolver $resolver)
      */
     public function getParent()
     {
-        return 'choice';
+        return class_exists('Symfony\Component\Form\Extension\Core\Type\ChoiceType') ?
```

Comment:
Why use this test instead of the one I provided you in the issue ? I am really not sure `ChoiceType` does not exist in sf < 2.8.